command/volume_register_csi.go

```diff
@@ -0,0 +1,44 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl"
+	"github.com/hashicorp/hcl/hcl/ast"
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/helper"
+)
+
+func (c *VolumeRegisterCommand) csiRegister(client *api.Client, ast *ast.File) int {
+	vol, err := csiDecodeVolume(ast)
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error decoding the volume: %s", err))
```

Comment:
Minor wording nitpick: maybe `"Error decoding the volume definition"` would be more clear to operators that the problem is at their end?

Reply:
fixed in ccace89da